src/insns/cadd_32bit.adoc

```diff
@@ -0,0 +1,65 @@
+<<<
+
+[#CADDI,reftext="CADDI"]
+==== CADDI
+
+See <<CADD>>.
+
+[#CADD,reftext="CADD"]
+==== CADD
+
+include::new_encoding_note.adoc[]
+
+ifdef::cheri_v9_annotations[]
+NOTE: *CHERI v9 Note:* these instructions were called CINCOFFSET and
+ CINCOFFSETIMM.
```

Comment:
Please change the outdated names here.

Reply:
those are the old names - that's the point of the comment